benchmark/generate_image_repo_parallel.py

```
@@ -1,21 +1,39 @@
     img = Image.fromarray(noise)
+
+    # Add index text if requested
+    if show_index:
+        # Make the image a black background
+        img = Image.new("RGB", image_size, (0, 0, 0))
+        draw = ImageDraw.Draw(img)
+        # Use a simple font and size that scales with image
+        font_size = min(image_size) // 10
+        try:
+            font = ImageFont.truetype("arial.ttf", font_size)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)